app/src/components/notifier/Notifier.js

```diff
@@ -8,47 +9,45 @@ const styles = {
     }
 };
 
-let openSnackbarFn;
+const AutoHideDuration = 10000;
 
 class Notifier extends React.Component {
+
   state = {
-    open: false,
-    message: '',
+     open: false,
   };
 
   componentDidMount() {
-    openSnackbarFn = this.openSnackbar;
   }
 
-  openSnackbar = ({ message }) => {
+  componentWillReceiveProps(nextProps) {
     this.setState({
-      open: true,
-      message,
+      open:(+new Date() - nextProps.notify.timestamp) < AutoHideDuration,
```

Comment:
it's worth commenting this line... I admit that the `+new Date()` is awkward looking otherwise :joy:

ex:
```//grab an instant JS timestamp to compare to time of last notify message; if within autohide duration, set open to true```